buildDocs.fsx

```diff
@@ -0,0 +1,1456 @@
+// Generate and Release Docs
```

Comment:
1456 lines of code for build script beg for more explanation then 1 sentence in commit message. It feels like inlined FAKE.

Reply:
The idea is to squash this as a single commit, so we can capture what it takes to add doc infrastructure to a project, but I'm open to other suggestion.

It contains many inlined portions of FAKE as it states in the following lines.

This way, we can build and release the docs with a single command line `fsi buildDocs.fsx ReleaseDocs`

Reply:
I've splitted the file in two, it's more readable now.